vignettes/rOpenscmRunner.Rmd

```diff
@@ -0,0 +1,162 @@
+
+```r
+install.packages("rOpenscmRunner")
+```
+
+Additionally, you will need a Python installation and the `openscm-runner` Python package installed, as well as some
+climate models. If you don't have a Python installation so far, you can find one at
+[anaconda](https://www.anaconda.com/products/distribution). Use the package management of your Python distribution
+to install `openscm-runner` with all dependencies for the models. If you use `pip`, run from the command line:
+
+```bash
+$ pip install --user openscm-runner[models]
+```
+
+If you want to run the MAGICC model, you also have to install it. Download the most recent version from
+[magicc.org](https://magicc.org/download/magicc7). Extract it to a location suitable to you, and copy the default
+configuration from the `run/defaults/` folder into the `run/` folder.
+
+## Setup
+
+FaIR and Cicero-SCM are installed via the Python package management or included in `openscm-runner`, so
+it knows where to finde them. However, MAGICC is installed separately, so you have to tell `rOpenscmRunner` where to
+find it using the `rOpenscmRunner::setup` function:
+
+```{r}
+rOpenscmRunner::setup(magiccExecutable7 = "/path/to/magicc/bin/magicc")
+```
+If you are on windows, you have to give the full path to the `magicc.exe` file in the MAGICC installation in the
+`bin/` folder. On Linux, give the full path to the `magicc` file in the `bin/` folder in the MAGICC installation.
+
+Using the same mechanism, you can also configure other parts of the MAGICC integration, but they are not required:
+```{r}
+rOpenscmRunner::setup(magiccWorkerNumber = 4, magiccWorkerRootDir = "/path/to/temp/folder")
+```
+The default for the number of workers is to use as many workers as there are processors in your system. This should
+be a sensible default on laptops and workstations, but probably needs manual setup on shared machines like compute
+clusters.
+The default for the temporary folder is to use your system's default temporary folder location. You might want to
+specify another temporary folder here, for example to take advantage of a fast in-memory file system. Note however
+that you are responsible to create and clean up the temporary folder yourself if you do specify one.
+
+All settings supplied via `rOpenscmRunner::setup()` are remembered for the duration of your current R session. If you
+don't want to call `rOpenscmRunner::setup()` in each session, you can also supply these settings via environment
+variables, check the documentation of `rOpenscmRunner::setup()` for details.
+
+## Usage
+
+To run simple climate model(s), use the `rOpenscmRunner::run()` function. It takes the emission scenario(s) as a
+`data.frame` in a format like the IAMC format for scenarios. In particular, it is a `data.frame` where each row is
+a time series and the columns are `model` for the model that produced the emissions scenario, `scenario` for a moniker
```

Comment:
I've never heard of a moniker, but you just mean a nickname here, right?

Reply:
Yeah, probably "scenario name" would be better here.